app/workers/bgs/submit_form674_job.rb

```diff
@@ -11,6 +11,10 @@ class Invalid674Claim < StandardError; end
 
     sidekiq_options retry: false
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Yea, we are supposed to go back and remove flippers.... I find that we often do not (myself included) so that had not been something I considered. If that is the plan I can see your reasoning. I still think it could be refactored in a way to satisfy both the removal of the flipper and the 2 rubocop errors, but it is not a huge deal or worth the time probably, especially since getting this out ASAP is a priority!

Reply:
Yeah, my goal is to make [the removal ticket](https://app.zenhub.com/workspaces/benefits-dependents-experience-6459139e7d64bf0015f6af13/issues/gh/department-of-veterans-affairs/va.gov-team/65763) when we create the PR, helps keep us a bit cleaner.